### ISSUE TITLE
Trait method boot has not been applied

### ISSUE THREAD

**Issue by widlturk** [spatie/laravel-medialibrary#7]
Trait method boot has not been applied, because there are collisions with other trait methods on App\Models\User 

use Spatie\MediaLibrary\MediaLibraryModel\MediaLibraryModelInterface;
use Spatie\MediaLibrary\MediaLibraryModel\MediaLibraryModelTrait;

use Zizaco\Entrust\Traits\EntrustUserTrait;
use Laracasts\Presenter\PresentableTrait;

class User extends Model implements AuthenticatableContract, CanResetPasswordContract, MediaLibraryModelInterface {
    use Authenticatable, CanResetPassword, PresentableTrait, EntrustUserTrait;
    use MediaLibraryModelTrait;

**Comment by widlturk** [comment]
#as a solution

##I add the following functions to the model file

```php
public static function boot()
    {
        parent::boot();

        self::deleting(function (MediaLibraryModelInterface $subject) {

            foreach ($subject->media()->get() as $media) {
                MediaLibrary::remove($media->id);
            }
        });
    }
	
	public function getImageProfileProperties()
    {
        return [
            'thumb' => ['w'=>200, 'h'=>200]
        ];
    }

**Comment by freekmurze** [comment]
There's no need to add that method in your model file: I corrected the problem in version 1.5.5 of the package.

**Comment by widlturk** [comment]
thanks
